ts/components/ConnectionBar.tsx

```diff
@@ -33,6 +33,7 @@ const styles = StyleSheet.create({
     marginRight: customVariables.fontSizeBase / 2
   },
   text: {
+    fontFamily: "RobotoMono-Regular",
```

Comment:
we should have this handled in native base theme - can you try importing `<Text>` from `native-base` instead of `react-native` ? - cc @fpersico

Reply:
The default <Text> font is Titillium not Roboto.

Reply:
Then I guess if we switch to NB's `Text` we should see the message in Titillium in both Android and iOS, let's see if it looks good?

Reply:
Ok, I will try to switch.

Reply:
By using the Text component imported from _native-base_ the problem is solved and it looks good on both OS.